net-p2p/go-ipfs-bin/go-ipfs-bin-0.4.22.ebuild

```diff
@@ -17,7 +17,12 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="-* ~amd64 ~arm ~x86"
 
-RDEPEND="sys-fs/fuse"
+RDEPEND="
+	acct-group/syncthing
+	acct-user/syncthing
```

Comment:
Small error here, it should be ipfs not syncthing :p

Reply:
Oh noes! Now you know my secret... ;-)

Okay, fixed.